package.json

```diff
@@ -105,7 +105,7 @@
         "vite-plugin-eslint": "1.8.1"
     },
     "dependencies": {
-        "@frontify/fondue-tokens": "3.1.0",
+        "@frontify/fondue-tokens": "^3.2.0",
```

Comment:
```suggestion
        "@frontify/fondue-tokens": "3.2.0",
```
Please pin the dependency to avoid any issue :)

Reply:
Thanks a lot! :D